src/Features/Core/Portable/PullMemberUp/QuickAction/InterfacePullerWithQuickAction.cs

```diff
@@ -0,0 +1,27 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.  
+
+using Microsoft.CodeAnalysis.Shared.Utilities;
+
+namespace Microsoft.CodeAnalysis.PullMemberUp.QuickAction
+{
+    internal class InterfacePullerWithQuickAction : AbstractMemberPullerWithQuickAction
+    {
+        protected override bool IsDeclarationAlreadyInTarget(
+            INamedTypeSymbol targetSymbol,
+            ISymbol userSelectedNodeSymbol)
+        {
+            var allMembers = targetSymbol.GetMembers();
+            var comparer = SymbolEquivalenceComparer.Instance;
+            foreach (var interfaceMember in allMembers)
+            {
+                var implementationOfMember = userSelectedNodeSymbol.ContainingType.FindImplementationForInterfaceMember(interfaceMember);
```

Comment:
rename all instances of userSelectedNodeSymbol to 'selectedMember'.  all instances of targetTypeSymbol to 'destinationType' plz.